openelis/src/MessageResources_pt.properties

```diff
@@ -71,7 +72,7 @@ errors.prefix=<LI>
 errors.header=<UL>
 AIDSStage.adultA=1 \= Stade A (Adulto)
 answer.yes=1 \= Sim
-answer.yesNo.description=1 \= Sim, 2 \= N\u00e3o
+answer.yesNo.description=1 \= Sim, 2 \= n�o
```

Comment:
Does this render correctly in OpenELIS? 

I remember we were having issues with special character encodings not rendering correctly hence we used html tags